scripts/run_lighthouse_tests.py

```diff
@@ -175,20 +175,16 @@ def run_lighthouse_checks(lighthouse_mode, shard):
         sys.exit(1)
 
 
-def main(args=None):
+def main(args=None) -> None:
     """Runs lighthouse checks and deletes reports."""
     parsed_args = _PARSER.parse_args(args=args)
 
     if parsed_args.mode == LIGHTHOUSE_MODE_ACCESSIBILITY:
         lighthouse_mode = LIGHTHOUSE_MODE_ACCESSIBILITY
         server_mode = SERVER_MODE_DEV
-    elif parsed_args.mode == LIGHTHOUSE_MODE_PERFORMANCE:
+    else:
         lighthouse_mode = LIGHTHOUSE_MODE_PERFORMANCE
         server_mode = SERVER_MODE_PROD
-    else:
-        raise Exception(
-            'Invalid parameter passed in: \'%s\', please choose'
-            'from \'accessibility\' or \'performance\'' % parsed_args.mode)
```

Comment:
Since we tell `argparse` that the only valid options for `--mode` are `accessibility` and `performance`, this `else` block will never run:

https://github.com/oppia/oppia/blob/feeb86b15c13b79ee5752ab5debce9c5dd73db96/scripts/run_lighthouse_tests.py#L62-L64